system_tests/bigquery.py

```diff
@@ -602,3 +613,18 @@ def test_dump_table_w_public_data(self):
         dataset = Config.CLIENT.dataset(DATASET_NAME, project=PUBLIC)
         table = dataset.table(TABLE_NAME)
         self._fetch_single_page(table)
+
+    def test_create_table_nested_schema(self):
+        TABLE_NAME = 'test_table'
```

Comment:
But that's because we've [disabled `invalid-name`][1] for tests. We can / should re-enable it? Though @lukesneeringer seems to be dumping Pylint for `flake8`

[1]: https://github.com/GoogleCloudPlatform/google-cloud-python/blob/cba8b9f18dda1e4510e5c3990cf84987f0e4b6ff/scripts/run_pylint.py#L57